### PR TITLE
Store certificate observations.

### PR DESCRIPTION
**Pull request by laiqu**

### CODE REVIEW

File: python/ct/client/db/sqlite_cert_db.py [hunk 1]
```diff
@@ -100,6 +107,16 @@ def __store_cert(self, cert, index, log_key, cursor):
...
+        if cert.observations:
+            for observation in cert.observations:
+                try:
+                    cursor.execute("INSERT INTO observations(log, cert_id,"
+                                   "description, reason, details)"
+                                   "VALUES(?, ?, ?, ?, ?)",
+                                   (log_key, index, observation.description,
+                                    observation.reason, "%s"%observation.details))
+                except:
+                    pass
```
Comment (eranmes) [inline python/ct/client/db/sqlite_cert_db.py]: Log? Somehow indicate the insertion failed?

Comment (eranmes) [inline python/ct/client/db/sqlite_cert_db.py]: Log? Somehow indicate the insert has failed?

**Comment by eranmes** [comment]
You should handle DB insertion failure. Other than that, looks good.

**Comment by laiqu** [comment]
Rewrote most of the pull request, because it was seriously outdated :(

File: python/ct/client/db/cert_desc.py [hunk 2]
```diff
@@ -48,6 +48,14 @@ def from_cert(certificate):
 
     proto.sha256_hash = hashlib.sha256(proto.der).digest()
 
+    for obs in observations:
```
Comment (eranmes) [inline python/ct/client/db/cert_desc.py]: nit: obs -> observation

**Comment by eranmes** [comment]
LGTM

**Pull request merged**